models/constant.py

```diff
@@ -0,0 +1,27 @@
+CRF_MODEL_TYPE = 'crf'
```

Comment:
Put comments throughout the file about what each and every constant means.  No one gets an idea about what city_o means